### ISSUE TITLE
baseVisitor and Listener not generated

### ISSUE THREAD

**Issue by mjkkirschner** [tunnelvisionlabs/antlr4ts#258]
my command looks something like:

```
    "antlr4ts": "node ./node_modules/.bin/antlr4ts -visitor ./test.g4 -o ./antlrStuff -Xlog"

```

I get this in the console:

```
Generating file ---./testLexer.ts' for grammar './test.g4'
Generating file '---./testLexer.tokens' for grammar './test.g4'
Generating file '---./testParser.ts' for grammar './test.g4'
Generating file '---./testListener.ts' for grammar './test.g4'
Generating file '---./testVisitor.ts' for grammar './test.g4'
Generating file '---./test.tokens' for grammar './test.g4'
```
There are no baseVisitor or baseListener classes generated. Should these be getting generated or am I mistaken?

My grammar is something like:

```
grammar test;


parse
  :  exp EOF 
  ;

exp
  :  addExp
  ;

addExp
  :  mulExp (('+' | '-') mulExp)*
  ;

mulExp
  :  unaryExp (('*' | '/') unaryExp)*
  ;

unaryExp
  :  '-' atom 
  |  atom
  ;

atom
  :  Number
  |  '(' exp ')' 
  ;

Number
  :  ('0'..'9')+ ('.' ('0'..'9')+)?
  ;
```

**Comment by sharwell** [comment]
There is no need for a base visitor or listener in TypeScript. The removal of these unnecessary types is in #196, and it was implemented in #200.

* In any case where you planned to have `extends testBaseListener`, use `implements testListener` instead.
* In any case where you planned to have `extends testBaseVisitor<T>`, use `extends AbstractParseTreeVisitor<T> implements testVisitor<T>` instead.

**Comment by mjkkirschner** [comment]
@sharwell thanks
